hypothesis-python/src/hypothesis/strategies.py

```diff
@@ -1047,6 +1049,12 @@ def builds(
     value :const:`hypothesis.infer` as a keyword argument to
     builds, instead of a strategy for that argument to the callable.
 
+    If the callable is a class defined with :pypi:`attrs`, missing required
+    arguments may be inferred from the type, converter, or validator (for
```

Comment:
I wonder if it would be better to be no-specific about this and just say "on a best-effort basis"? The details are kinda uninteresting - either it works and people barely have to know it's there, or it doesn't and people will write their own strategy here rather than tinkering with their class definitions.